.travis.yml

```diff
@@ -45,7 +45,13 @@ jobs:
       script: make build
       os: osx
     - stage: test
-      script: true
+      script:
+        - go test -v $(go list ./... | grep -vE "vendor|cli|utils")
```

Comment:
More fine-grained control of which tests to run. This passes.
The utils fails cause of the dependency upon /bin/true, which is at a different location on macOS. This could be handled I guess, if you want.
Also, the cli tests are dropped, but run below as a tautology.